### ISSUE TITLE
[AIR Output] Formatting issues for training output

### ISSUE THREAD

**Issue by scottsun94** [ray-project/ray#36755]
### What happened + What you expected to happen

Run the torch training example in ray air get started page: https://docs.ray.io/en/latest/ray-air/getting-started.html

1. Some lines of output are indented unexpectedly. Is there a reason for that?
<img width="1425" alt="Screenshot 2023-06-22 at 10 52 37 PM" src="https://github.com/ray-project/ray/assets/9677264/ead092a5-6319-42d0-b22f-90ac5a4b893f">

2. Can we add a blank line before printing the results of a new iteration so that users can visually distinguish them more easily?
```
Training finished iteration 1 at 2023-06-22 22:44:43. Total running time: 8s
╭─────────────────────────────────────────────────╮
│ Training result                                 │
├─────────────────────────────────────────────────┤
│ time_this_iter_s                        5.42888 │
│ time_total_s                            5.42888 │
│ training_iteration                            1 │
│ config/train_loop_config/batch_size         128 │
│ config/train_loop_config/lr               0.001 │
│ config/train_loop_config/num_epochs          20 │
│ config/train_loop_config/num_features        30 │
│ loss                                         50 │
╰─────────────────────────────────────────────────╯
Training saved a checkpoint for iteration 1 at: /home/ray/ray_results/TorchTrainer_2023-06-22_22-44-34/TorchTrainer_08987_00000_0_2023-06-22_22-44-35/checkpoint_000000
Training finished iteration 2 at 2023-06-22 22:44:43. Total running time: 8s
╭─────────────────────────────────────────────────╮
│ Training result                                 │
├─────────────────────────────────────────────────┤
│ time_this_iter_s                        0.12604 │
│ time_total_s                            5.55492 │
│ training_iteration                            2 │
│ config/train_loop_config/batch_size         128 │
│ config/train_loop_config/lr               0.001 │
│ config/train_loop_config/num_epochs          20 │
│ config/train_loop_config/num_features        30 │
│ loss                                         50 │
╰─────────────────────────────────────────────────╯
Training saved a checkpoint for iteration 2 at: /home/ray/ray_results/TorchTrainer_2023-06-22_22-44-34/TorchTrainer_08987_00000_0_2023-06-22_22-44-35/checkpoint_000001
Training finished iteration 3 at 2023-06-22 22:44:43. Total running time: 8s
╭─────────────────────────────────────────────────╮
│ Training result                                 │
├─────────────────────────────────────────────────┤
│ time_this_iter_s                        0.13403 │
│ time_total_s                            5.68895 │
│ training_iteration                            3 │
│ config/train_loop_config/batch_size         128 │
│ config/train_loop_config/lr               0.001 │
│ config/train_loop_config/num_epochs          20 │
│ config/train_loop_config/num_features        30 │
│ loss                                         50 │
╰─────────────────────────────────────────────────╯
Training saved a checkpoint for iteration 3 at: /home/ray/ray_results/TorchTrainer_2023-06-22_22-44-34/TorchTrainer_08987_00000_0_2023-06-22_22-44-35/checkpoint_000002
```

3. Can we also add a blank line between the last iteration result and the summary of the training? Can we also rephrase it to "Training completed after 20 iterations at 2023-06-22 22:44:47. Total running time: 12s"? (removing the 2nd `training`)
```
Training finished iteration 20 at 2023-06-22 22:44:45. Total running time: 10s
╭─────────────────────────────────────────────────╮
│ Training result                                 │
├─────────────────────────────────────────────────┤
│ time_this_iter_s                        0.12567 │
│ time_total_s                            7.84838 │
│ training_iteration                           20 │
│ config/train_loop_config/batch_size         128 │
│ config/train_loop_config/lr               0.001 │
│ config/train_loop_config/num_epochs          20 │
│ config/train_loop_config/num_features        30 │
│ loss                                         50 │
╰─────────────────────────────────────────────────╯
Training saved a checkpoint for iteration 20 at: /home/ray/ray_results/TorchTrainer_2023-06-22_22-44-34/TorchTrainer_08987_00000_0_2023-06-22_22-44-35/checkpoint_000019
Training completed training after 20 iterations at 2023-06-22 22:44:47. Total running time: 12s
Last result: {'loss': 50.0, 'timestamp': 1687499085, 'time_this_iter_s': 0.12567496299743652, 'should_checkpoint': True, 'done': True, 'training_iteration': 20, 'trial_id': '08987_00000', 'date': '2023-06-22_22-44-45', 'time_total_s': 7.848382234573364, 'pid': 15525, 'hostname': 'g-e217ea33a67810001', 'node_ip': '10.0.0.199', 'config': {'train_loop_config': {'batch_size': 128, 'num_epochs': 20, 'num_features': 30, 'lr': 0.001}}, 'time_since_restore': 7.848382234573364, 'iterations_since_restore': 20, 'experiment_tag': '0'}
```

### Versions / Dependencies

nightly

### Reproduction script

https://docs.ray.io/en/latest/ray-air/getting-started.html

### Issue Severity

Low: It annoys or frustrates me.

**Comment by krfricke** [comment]
Fixing 2+3 in #36765.

For 1, this is a datasets problem (probably tqdm-related) and probably related to #34114. I'll look into disabling it again, but was unsuccessful last time...
